bigquery/google/cloud/bigquery/client.py

```diff
@@ -1491,6 +1492,15 @@ def load_table_from_dataframe(
                 column names matching those of the dataframe. The BigQuery
                 schema is used to determine the correct data type conversion.
                 Indexes are not loaded. Requires the :mod:`pyarrow` library.
+            parquet_compression (str):
+                 The compression method to use if intermittently serializing
```

Comment:
Let's mark this as `[Beta]`. I'm hesitant to call this a GA feature since it's so dependent on an implementation detail and third-party dependencies.